crates/nu-cli/src/util.rs

```diff
@@ -286,6 +286,34 @@ pub fn eval_source(
     true
 }
 
+pub fn eval_source2(
```

Comment:
Seems like the intent of `eval_source2` is to strip away the printing functionality from the original `eval_source()` and the display hook and to be used to evaluate the commands inside the pager, right? I'm thinking that a) it could probably use a better name and b) shouldn't it be a part of the pager instead of changing Nushell internals?

Reply:
> Seems like the intent of eval_source2 is to strip away the printing functionality from the original eval_source()

Yes

>  a) it could probably use a better name and

Not the worth name though :)

> b) shouldn't it be a part of the pager instead of changing Nushell internals?

It'd better be, if it can be moved it should.
*I'll check it.

Reply:
moved.